kodilla-jdbc/src/test/resources/task12_4

```diff
@@ -0,0 +1,14 @@
+  SELECT L.NAME, I.SUMMARY, I.DESCRIPTION
+      FROM ISSUESLISTS L, ISSUES I
+      WHERE L.ID = I.ISSUESLIST_ID;
+
+  SELECT U.FIRSTNAME, U.LASTNAME, I.SUMMARY, I.DESCRIPTION
+      FROM USERS U, ISSUES I
+      WHERE U.ID = I.USER_ID_ASSIGNEDTO;
+
+  SELECT U.ID, U.FIRSTNAME, U.LASTNAME,
+      COUNT(I.DESCRIPTION) AS TASKS_NUMBER
```

Comment:
lepiej zawsze zliczać po kluczu głównym - akurat dałeś unique na description, ale to jakos tak obco wygląda
no i zamiast TASKS_NUMBER dałbym nazwę TASKS_COUNT